src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/ReconciliationServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.blobrouter.reconciliation.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.statements.SupplierStatementRepository;
+import uk.gov.hmcts.reform.blobrouter.data.reconciliation.statements.model.NewEnvelopeSupplierStatement;
+import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidSupplierStatementException;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.in.Envelope;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.model.in.SupplierStatement;
+
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.util.UUID;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class ReconciliationServiceTest {
+
+    private ReconciliationService service;
+
+    @Mock
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    @Mock
+    private SupplierStatementRepository repository;
+
+    @BeforeEach
+    void setUp() {
+        service = new ReconciliationService(repository, objectMapper);
+    }
+
+    @Test
+    void should_save_the_supplier_statement() throws Exception {
```

Comment:
should_save_the_supplier_statement when part missing ?

Reply:
updated the test